test/integration.js

```diff
@@ -0,0 +1,146 @@
+'use strict';
+
+const assert = require('assert');
+const blc = require('broken-link-checker');
+const test = require('selenium-webdriver/testing');
+const webdriver = require('selenium-webdriver');
+
+const timeOut = 35000;
+
+const baseURL = process.env.VS_TEST_URL || 'https://viewsourceconf-stage.us-west.moz.works';
+console.log(`Running integration tests against ${baseURL}`)
+
+describe('The site...', function() {
+    this.timeout(timeOut);
+    const url = baseURL;
+
+    it('should not have any broken links', function(done) {
+        const options = {
+            // https://github.com/stevenvachon/broken-link-checker
+            excludedKeywords: ['/2015', 'livereload'],
+            filterLevel: '3',
+        };
+
+        const errors = {};
+
+        const siteChecker = new blc.SiteChecker(options, {
+            link(result) {
+                if (result.broken) {
+                    errors[result.base.resolved] = errors[result.base.resolved] || [];
+                    errors[result.base.resolved].push(
+                        `${result.url.resolved} (${blc[result.brokenReason]})`
+                    );
+                }
+            },
+            site() {
+                if (Object.keys(errors).length > 0) {
+                    const msg = `Site has broken links: \n ${JSON.stringify(errors)}`;
+                    return done(new Error(msg));
+                }
+                return done();
+            },
+        });
+
+        siteChecker.enqueue(url, url);
+    });
+});
+
+test.describe('The landing page...', function() {
+    this.timeout(timeOut);
+    const url = baseURL;
+    const expectedTitle = 'View Source Conference 2016';
+
+    test.it(`should be called "${expectedTitle}"`, function(done) {
+        const driver = new webdriver.Builder()
+            .forBrowser('firefox')
```

Comment:
Rather than specify the browser directly in each test, is there a way to pass this via the command line so the test suite can be run in multiple browsers? Or is this already possible?